tests/touch

```diff
@@ -5,4 +5,5 @@
 
 import os, sys, time
 with open(sys.argv[1], "a") as f: f.write(" ")
+with open("tmp/touch.exe", "w") as f: f.write(os.readlink("/proc/self/exe"))
```

Comment:
This directory [does not exist](https://github.com/martinpitt/fatrace/actions/runs/16517862122/job/46712607435?pr=58#step:4:13). Please use `AUTOPKGTEST_TMP` if possible.